scripts/include-sf.js

```diff
@@ -4,14 +4,15 @@ const path = require('path');
 const shelljs = require('shelljs');
 const fs = require('fs');
 
-const sfUnixPath = 'node_modules/@salesforce/cli/bin/run';
+const sfGlobalPath = path.join(shelljs.exec('npm list -g --depth 0 | head -1'), 'node_modules', '@salesforce', 'cli');
+
+// Copy sf to the bin/cli dir of sfdx
+shelljs.mv('-f', sfGlobalPath, path.join(process.cwd(), 'bin', 'sf-cli'));
+
+const sfUnixPath = 'bin/sf-cli/bin/run';
```

Comment:
I think if you drop the `bin/` part you can undo this change from a couple days ago: https://github.com/salesforcecli/sfdx-cli/pull/276/files